.github/workflows/appstore-build-publish.yml

```diff
@@ -0,0 +1,169 @@
+# This workflow is provided via the organization template repository
+#
+# https://github.com/nextcloud/.github
+# https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization
```

Comment:
Everything here is just an exact copy of what's at the template repo. I expected some lines will be changed to make the template suitable for Tables specifically? :thinking: